terraform/test-fixtures/apply-destroy-nested-module-with-attrs/middle/bottom/bottom.tf

```diff
@@ -1 +1,5 @@
-variable "bottom_param" {}
+variable bottom_param {}
+
+resource "null_resource" "bottom" {
+  value = "${var.bottom_param}"
```

Comment:
Just a question to help me form good reproductions in the future: What's this `value` on `null_resource`? Can you just put whatever keys you want on `null_resource`, or is `value` something special?